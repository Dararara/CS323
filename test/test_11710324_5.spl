int test_1_m01(int a, int b)
{
  int code[10], new;
  code[10 = 10;
  0w0 = '\xjd'
  return code[new[0];
}