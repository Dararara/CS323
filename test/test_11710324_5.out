Error type B at Line 4: Missing closing ']'
Error type A at Line 5: unknown lexeme 0w0
Error type A at Line 5: unknown lexeme '\xjd'
Error type B at Line 0: Missing semicolon ';'
Error type B at Line 6: Missing closing ']'
